gpkit/tests/t_constraints.py

```diff
@@ -14,6 +14,24 @@
 class TestConstraint(unittest.TestCase):
     """Tests for Constraint class"""
 
+    def test_bad_elements(self):
+        x = Variable("x")
+        with self.assertRaises(ValueError):
+            _ = Model(x, [x == "A"])
+        with self.assertRaises(ValueError):
+            _ = Model(x, [x >= 1, x == "A"])
+        with self.assertRaises(ValueError):
+            _ = Model(x, [x >= 1, x == "A", x >= 1, ])
+        with self.assertRaises(ValueError):
+            _ = Model(x, [x == "A", x >= 1])
+        v = VectorVariable(2, "v")
+        with self.assertRaises(ValueError):
+            _ = Model(x, [v == "A"])
+        with self.assertRaises(ValueError):
+            _ = Model(x, [v <= ["A", "B"]])
+        with self.assertRaises(ValueError):
+            _ = Model(x, [v >= ["A", "B"]])
```

Comment:
These tests all involve string comparison; I also want to see the much more interesting case of `VectorVariable(2, "v") == Variable("x")`